subworkflows/nf-core/variantcalling/freebayes/main.nf

```diff
@@ -31,10 +32,9 @@ workflow RUN_FREEBAYES {
     TABIX_VC_FREEBAYES(BCFTOOLS_SORT.out.vcf)
 
     // Only when using intervals
-    BGZIP_VC_FREEBAYES(freebayes_vcf_out.intervals)
-
-    CONCAT_FREEBAYES(
-        BGZIP_VC_FREEBAYES.out.output
+    BCFTOOLS_SORT_INTERVAL_VCFS(freebayes_vcf_out.intervals)
```

Comment:
Don't think we need two BCFtools sortings here, since the entire output of freebayes gets sorted now.

I would instead propose:
```
BCFTOOLS_SORT(FREEbayes.out.vcf)
BCFTOOLS_SORT.out.vcf.branch{
 intervals:    it[0].num_intervals > 1
            no_intervals: it[0].num_intervals <= 1
        }.set{bcftools_vcf_out}
        
        TABIX_VC_FREEBAYES(bcftools_vcf_out.no_intervals)
        MERGEFREEBAYES(bcftools_vcf_out.intervals.map{} etc )
       ```